### PR TITLE
{bio}[foss-2015b] Trim_Galore 0.4.1

### PR DESCRIPTION
**Pull request by verdurin**
Needs #2568 and #2569

### CODE REVIEW

**Comment by boegel** [comment]
Jenkins: ok to test

**Comment by boegel** [comment]
```
EasyBuildError: 'Irresolvable dependencies encountered: FastQC/0.11.4-foss-2015b'
```

related: #2267

**Comment by boegel** [comment]
Jenkins: test this please

**Comment by verdurin** [comment]
@boegel - the failure is just because it requires the `fastqc` update, yes?

File: easybuild/easyconfigs/t/Trim_Galore/Trim_Galore-0.4.1-foss-2015b.eb [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+
+toolchain = {'name': 'foss', 'version': '2015b'}
+
+sources = ['%(namelower)s_v%(version)s.zip']
+source_urls = ['http://www.bioinformatics.babraham.ac.uk/projects/%(namelower)s']
+
+install_cmd = "cp -a %(namelower)s_zip/* %(installdir)s && chmod +x %(installdir)s/%(namelower)s"
+
+dependencies = [
+    ('FastQC', '0.11.4'),
```
Comment (boegel) [inline easybuild/easyconfigs/t/Trim_Galore/Trim_Galore-0.4.1-foss-2015b.eb]: @verdurin this needs to be changed to something like

```python
('FastQC', '0.10.1', '-Java-1.8.0_74', True),
```

**Comment by migueldiascosta** [comment]
@verdurin, this PR is being closed for the following reason(s): no activity for > 6 months, obsoleted by more recent PRs.
Please don't hesitate to reopen this PR or add a comment if you feel this contribution is still relevant.
For more information on our policy w.r.t. closing PRs, see https://easybuild.readthedocs.io/en/latest/Contributing.html#why-a-pull-request-may-be-closed-by-a-maintainer

**Pull request closed without merging**